src/hooks/useCanInteractWithColony.ts

```diff
@@ -48,10 +48,24 @@ export const useCanInteractWithColony = (colony?: Colony): boolean => {
   if (!wallet || !colony) {
     return false;
   }
+
+  /*
+   * Check if connected to the same chain
+   */
   const [{ id: walletHexChainId }] = wallet.chains;
   const colonyChain = colony?.meta?.chainId || DEFAULT_NETWORK_INFO.chainId;
   const userWalletChain = parseInt(walletHexChainId.slice(2), 16);
-  return colonyChain === userWalletChain && canInteractWithNetwork;
+
+  /*
+   * Checking if watching (following) or not
+   */
+  const isWatching = !!(user?.watchlist?.items || []).find(
+    (item) => (item?.colony as Colony)?.colonyAddress === colony?.colonyAddress,
+  );
+
+  return (
+    colonyChain === userWalletChain && canInteractWithNetwork && isWatching
+  );
```

Comment:
I felt this check was relevant. Feel free to correct me if it is not the intended purpose of `useCanInteractWithColony`.

Reply:
Should we update this copy as well? e.g. connect wallet, register user, _join the colony_ ...

![colonyinteract](https://user-images.githubusercontent.com/64402732/211000912-db720fd4-3a3e-4a6c-a5cf-9082ff5d48d7.png)

Reply:
Yeah, good point. I did not even see that come up.